common/src/model_registration/register.py

```diff
@@ -129,7 +131,22 @@ def main(args):
             max_version = (max(models_list, key=lambda x: x.version)).version
             model_version = str(int(max_version) + 1)
     except Exception:
-        print(f"Error in listing versions for model {model_name}. Trying to register model with version '1'.")
+        print(
```

Comment:
Do we need this for non-registry operations?

Reply:
are we planning to remove registry_name from component field?

Reply:
Nope, we are not planning to move registry_name ! We will have to register in registry eventually using component, so not planning to remove it

Reply:
Sounds good